src/sections/home-text-with-images.liquid

```diff
@@ -54,10 +54,20 @@
   section[data-section-id="{{ section.id }}"] .custom-color {
     color: #fff;
   }
+
+  {% if section.settings.image_overlay_opacity != blank %}
+  .image-container::after {
+    opacity: {{ section.settings.image_overlay_opacity | times: 0.1 }};
+  }
+  {% endif %}
```

Comment:
@rdnvol please add specific selector for this section. If you don't do it, there will be a problem when you will add two or more sections on the page (settings from first section will overlap settings from other sections, and in other sections there will be incorrect display of this style property, you may try it by yourself). So what you need to do is - add `section[data-section-id="{{ section.id }}"]` (as it made for style above), to make your style "section specific"

Reply:
@rdnvol and also pls fix formatting here
```suggestion
  {% if section.settings.image_overlay_opacity != blank %}
    .image-container::after {
      opacity: {{ section.settings.image_overlay_opacity | times: 0.1 }};
    }
  {% endif %}
```